### PR TITLE
Fix feedforward classes to be unitless

### PR DESCRIPTION
**Pull request by auscompgeek**

### CODE REVIEW

**Comment by virtuald** [comment]
I'm not sure this is a good idea, because the units interact with volts (kv, ka) internally, unlike the controller classes.

**Comment by auscompgeek** [comment]
Sure, but what units do you use to measure the distance a flywheel spins?

The Java API makes these unitless because the units do indeed cancel out.

**Comment by virtuald** [comment]
I would be fine with you adding a scalar version that has the original name. I don't really want to remove them at this time, because we've already broken users enough times this season.

File: wpilib/controller/__init__.py [hunk 1]
```diff
@@ -3,22 +3,22 @@
...
     ProfiledPIDController,
     RamseteController,
+    SimpleMotorFeedforward,
     SimpleMotorFeedforwardFeet,
     SimpleMotorFeedforwardMeters,
 )
 
 __all__ = [
     "ArmFeedforward",
-    "ElevatorFeedforwardFeet",
```
Comment (virtuald) [inline wpilib/controller/__init__.py]: Should still be in all.

Comment (auscompgeek) [reply wpilib/controller/__init__.py]: Do we really care about documenting these? I don't think we want to actively encourage users to use them if we're going to remove them next season.

**Pull request merged**